core/src/main/resources/schema/app/20200228140101_CB-5752_rename_template.sql

```diff
@@ -0,0 +1,11 @@
+-- // CB-5752 OpDB 7.1.0 cluster definitions out of sync for AWS and Azure
+-- Migration SQL that makes the change goes here.
+
+UPDATE clustertemplate SET name = '7.1.0 - Operational Database with SQL for AWS' WHERE status = 'DEFAULT' AND name = '7.1.0 - Operational Database with Phoenix for AWS';
```

Comment:
Should be extended to set the `7.1.0 - Operational Database: Apache HBase` blueprint to `DEFAULT_DELETED`

Reply:
I have added the update statement for that now.

Reply:
thanks @stoty